core/dbt/cli/main.py

```diff
@@ -189,6 +189,7 @@ def cli(ctx, **kwargs):
 @p.exclude
 @p.full_refresh
 @p.include_saved_query
+@p.no_skip_on_failture
```

Comment:
```suggestion
@p.no_skip_on_failure
```

I think that this flag should be added to `dbt clone` on line 637 as well.